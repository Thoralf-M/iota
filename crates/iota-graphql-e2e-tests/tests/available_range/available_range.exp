processed 5 tasks

task 1, lines 7-29:
//# run-graphql
Response: {
  "data": {
    "availableRange": {
      "first": {
        "digest": "DdBQpdVrXM8NVmomVxsFaxJcuDfdQaMmSN9JDr9Xumcz",
        "sequenceNumber": 0
      },
      "last": {
        "digest": "DdBQpdVrXM8NVmomVxsFaxJcuDfdQaMmSN9JDr9Xumcz",
        "sequenceNumber": 0
      }
    },
    "first": {
      "digest": "DdBQpdVrXM8NVmomVxsFaxJcuDfdQaMmSN9JDr9Xumcz",
      "sequenceNumber": 0
    },
    "last": {
      "digest": "DdBQpdVrXM8NVmomVxsFaxJcuDfdQaMmSN9JDr9Xumcz",
      "sequenceNumber": 0
    }
  }
}

task 2, line 31:
//# create-checkpoint
Checkpoint created: 1

task 3, line 34:
//# create-checkpoint
Checkpoint created: 2

task 4, lines 37-59:
//# run-graphql
Response: {
  "data": {
    "availableRange": {
      "first": {
        "digest": "DdBQpdVrXM8NVmomVxsFaxJcuDfdQaMmSN9JDr9Xumcz",
        "sequenceNumber": 0
      },
      "last": {
        "digest": "2q9Vokrv1osF4LL8HfrjU68d6XNyi5QkyXDGe2x4EXPz",
        "sequenceNumber": 2
      }
    },
    "first": {
      "digest": "DdBQpdVrXM8NVmomVxsFaxJcuDfdQaMmSN9JDr9Xumcz",
      "sequenceNumber": 0
    },
    "last": {
      "digest": "2q9Vokrv1osF4LL8HfrjU68d6XNyi5QkyXDGe2x4EXPz",
      "sequenceNumber": 2
    }
  }
}
