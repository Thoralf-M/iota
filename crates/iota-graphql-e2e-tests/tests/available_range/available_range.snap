---
source: external-crates/move/crates/move-transactional-test-runner/src/framework.rs
---
processed 5 tasks

task 1, lines 9-31:
//# run-graphql
Response: {
  "data": {
    "availableRange": {
      "first": {
        "digest": "HA81rp38NCuAG3umu7XtxzZLVcWpJeTFuE33BCCq6Ztw",
        "sequenceNumber": 0
      },
      "last": {
        "digest": "HA81rp38NCuAG3umu7XtxzZLVcWpJeTFuE33BCCq6Ztw",
        "sequenceNumber": 0
      }
    },
    "first": {
      "digest": "HA81rp38NCuAG3umu7XtxzZLVcWpJeTFuE33BCCq6Ztw",
      "sequenceNumber": 0
    },
    "last": {
      "digest": "HA81rp38NCuAG3umu7XtxzZLVcWpJeTFuE33BCCq6Ztw",
      "sequenceNumber": 0
    }
  }
}

task 3, line 35:
//# create-checkpoint
Checkpoint created: 1

task 4, lines 37-47:
//# run-graphql
Response: {
  "data": {
    "availableRange": {
      "first": {
        "sequenceNumber": 0
      },
      "last": {
        "sequenceNumber": 1
      }
    }
  }
}

task 6, line 51:
//# create-checkpoint
Checkpoint created: 2

task 7, lines 53-63:
//# run-graphql
Response: {
  "data": {
    "availableRange": {
      "first": {
        "sequenceNumber": 0
      },
      "last": {
        "sequenceNumber": 2
      }
    }
  }
}

task 9, line 67:
//# create-checkpoint
Checkpoint created: 3

task 10, lines 69-79:
//# run-graphql
Response: {
  "data": {
    "availableRange": {
      "first": {
        "sequenceNumber": 1
      },
      "last": {
        "sequenceNumber": 3
      }
    }
  }
}

task 12, line 83:
//# create-checkpoint
Checkpoint created: 4

task 13, lines 85-95:
//# run-graphql
Response: {
  "data": {
    "availableRange": {
      "first": {
        "sequenceNumber": 2
      },
      "last": {
        "sequenceNumber": 4
      }
    }
  }
}

task 15, line 99:
//# create-checkpoint
Checkpoint created: 5

task 16, lines 101-121:
//# run-graphql
Response: {
  "data": {
    "availableRange": {
      "first": {
        "sequenceNumber": 3
      },
      "last": {
        "sequenceNumber": 5
      }
    },
    "first": {
      "digest": "HA81rp38NCuAG3umu7XtxzZLVcWpJeTFuE33BCCq6Ztw",
      "sequenceNumber": 0
    },
    "last": {
      "digest": "2QtPyuUTV9HN6wS2YUGop4yBYfRFqYwrKsC3fC8Q8ZeJ",
      "sequenceNumber": 5
    }
  }
}
